### PR TITLE
Use metal for caption and inlineLink

### LINKED ISSUE #555: Use metal for caption, timestamp, visited link
**Is your feature request related to a problem? Please describe.**
We want the text of:

- [ ] caption
- [ ] timestamp
- [ ] visited link

to be the shade [#6E6E73 metal](https://bbc.github.io/psammead/?path=/story/utilities-psammead-styles--colours) 

**Describe the solution you'd like**
1. css updated to use metal colour
2. Please add before and after screenshot comparisons.

**Testing notes**
* colour contrast checking

Dev insight: Unit / snapshot tests are fine.

**Additional context**

### PR DESCRIPTION
**Pull request by twinlensreflex**
Resolves #555 
(cloned https://github.com/bbc/psammead/pull/687 to rename branch)

**Overall change:** 
Changes `C_CLOUD_DARK` to `C_METAL` in `Caption text` and `InlineLink visited text`.
Timestamp has apparently been changed in another PR because it's already `C_METAL`.

**Code changes:**

- Changed `C_CLOUD_DARK` to `C_METAL` in Caption
- Updates Changelog & package.json & package_lock.json
- Changed `C_CLOUD_DARK` to `C_METAL` in InlineLink
- Updates Changelog & package.json & package_lock.json
- Removes `C_CLOUD_DARK` from Styles
- Updates Changelog & package.json & package_lock.json

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by eleanombre** [comment]
clone of https://github.com/bbc/psammead/pull/687

File: packages/components/psammead-caption/package.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-caption/README.md",
   "dependencies": {
     "@bbc/gel-foundations": "^3.0.0",
-    "@bbc/psammead-styles": "^0.3.2"
+    "@bbc/psammead-styles": "^1.0.1"
```
Comment (pjlee11) [inline packages/components/psammead-caption/package.json:21]: `"version": "1.1.7",` also needs updating to `"version": "1.1.8",` and `npm i` needs running to update `package-lock.json` version number

**Comment by pjlee11** [comment]
On npm registry it's only showing `psammead-styles@1.0.0` not `1.0.1` - https://www.npmjs.com/package/@bbc/psammead-styles

It's likely the the package.json wasn't updated so the publish npm didn't run, or the publish to npm failed

File: packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -15,7 +15,6 @@ exports[`Caption should render with some offscreen text 1`] = `
 .c0 {
   font-size: 0.9375rem;
   line-height: 1.125rem;
-  color: #757575;
```
Comment (pjlee11) [inline packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap:18]: This is highlighting that the Caption isn't pulling in the new `C_METAL` colour and the caption is actually losing it's grey colour. Quickly debugging this I realised that due to the mixup with `psammead-styles@1.0.1` not being in the `npm` registry that the package-lock for caption is wrong. 

To fix this checkout the branch, `cd packages/components/psammead-caption` then run `npm i` and the `package-lock,json` should update. Also remember to update the snapshots after the package-lock

File: packages/components/psammead-caption/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.8 | [PR#687](https://github.com/bbc/psammead/pull/687) Change text colour to `C_CLOUD_DARK` for `C_METAL` |
```
Comment (12) [inline packages/components/psammead-caption/CHANGELOG.md]:
```suggestion
| 1.1.8 | [PR#763](https://github.com/bbc/psammead/pull/763) Change text colour to `C_CLOUD_DARK` for `C_METAL` |
```

File: packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (pjlee11) [reply packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap:18]: Without package-lock update, black default browser colour (as per the snapshot output)
image

With new package-lock update, grey colour as expected. 
image

Comment (pjlee11) [reply packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap:18]: Confirmed this with @eleanombre, pushing the change now

**Review by pjlee11 — APPROVED** [review]
👍 Happy now the package-lock is up-to-date

**Review by oluoluoxenfree — APPROVED** [review]
LGTM!

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**